src/poliastro/twobody/orbit.py

```diff
@@ -550,25 +543,20 @@ def sample(self, values=None, method=mean_motion):
         <GCRS Coordinate ...>
         >>> iss.sample(10)  # doctest: +ELLIPSIS
         <GCRS Coordinate ...>
-        >>> iss.sample([0, 180] * u.deg)  # doctest: +ELLIPSIS
-        <GCRS Coordinate ...>
-        >>> iss.sample([0, 10, 20] * u.minute)  # doctest: +ELLIPSIS
-        <GCRS Coordinate ...>
-        >>> iss.sample([iss.epoch + iss.period / 2])  # doctest: +ELLIPSIS
-        <GCRS Coordinate ...>
 
         """
         if values is None:
-            return self.sample(100, method)
+            return self.sample(100)
```

Comment:
And therefore this `is values is None` block and the `else` that came after can be removed.